src/store/Workspaces/index.ts

```diff
@@ -288,6 +288,9 @@ export const actionCreators: ActionCreators = {
       await WorkspaceClient.restApiClient.stop(workspaceId);
     } catch (e) {
       dispatch({ type: 'RECEIVE_ERROR' });
+      if (e.message) {
+        throw new Error(`Failed to stop the workspace, ID: ${workspaceId}, ` + e.message);
```

Comment:
Probably workspace client needs to be fixed as well

@JPinkney Could you check what is the response from Che Server?
If it's something like `Failed to delete workspace since it needs to be stopped before` then workspace client should propagate it instead of `Request failed with status code 409`

Reply:
Looks like you can get better errors from che server so I've switched the error message to use that one